### PR TITLE
Adding tox to Python-RDC to make documenting the client very easy.

### PR DESCRIPTION
**Pull request by TimSimpsonR**
- Added some Sphinx docs.

### CODE REVIEW

**Comment by rnirmal** [comment]
+1 awesome job.

File: README.rst [hunk 1]
```diff
@@ -5,15 +5,17 @@ This is a client for the Reddwarf API. There's a Python API (the
...
 Command-line API
 ----------------
 
-TODO: Add docs
+To use the command line API, first log in using your user name, api key,
+tenant, and appropriate auth url.
+
+.. code-block:: bash
+
+    $ reddwarf-cli --username=jsmith --apikey=abcdefg --tenant=12345 --auth_url=http://reddwarf_auth:35357/v2.0/tokens auth login
```
Comment (cp16net) [inline README.rst:16]: i like the fact that you added this to the doc! thanks!

**Comment by cp16net** [comment]
+1

**Comment by jcru** [comment]
+1

**Comment by ed-** [comment]
+1

**Comment by imsplitbit** [comment]
+1

**Pull request merged**